src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
```

Comment:
Might not be accurate to import java generic, maybe you could import the specific classes utilised instead?

Reply:
Oops! I missed that. Will fix it.